graphrag_sdk/steps/qa_step.py

```diff
@@ -17,17 +15,18 @@ def __init__(
         self,
         chat_session: GenerativeModelChatSession,
         config: dict = None,
+        qa_prompt: str = None,
     ) -> None:
         self.config = config or {}
         self.chat_session = chat_session
+        self.qa_prompt = qa_prompt
 
     def run(self, question: str, cypher: str, context: str):
-
-        qa_prompt = GRAPH_QA_PROMPT.format(
+        qa_prompt = self.qa_prompt.format(
             context=context, cypher=cypher, question=question
```

Comment:
What guarantees do you have that `self.qa_prompt` contains a context, cypher and question place holders ?